iosx/src/flowmgr/AVSFlowManager.m

```diff
@@ -479,12 +479,13 @@ - (instancetype)init:(uint64_t)avs_flags
 #endif
 
 		g_Fm = self;
-	}
 
 #if !(TARGET_IPHONE_SIMULATOR)
-	_capturer = [[AVSCapturer alloc] init];
+		_capturer = [[AVSCapturer alloc] init];
```

Comment:
Some small whitespace error?

Reply:
I moved the allocing of _capturer to within the if (self) block because it would try to set _capturer if self is nil

Reply:
Ah, sorry, I missed the move of the end-bracket further down...